privacyidea/lib/crypto.py

```diff
@@ -404,7 +404,7 @@ def aes_encrypt_b64(key, data):
     """
     iv = geturandom(16)
     encdata = aes_encrypt(key, iv, data)
-    return base64.b64encode(iv + encdata)
+    return b64encode_and_unicode(iv + encdata)
```

Comment:
Looks sensible! Could you just adapt the docstring? :-)

Reply:
fixed in 21bdef5b